README.md

```diff
@@ -1,27 +1,25 @@
 [![](https://img.shields.io/static/v1?label=BY&message=RAMOUN&color=birghtgreen)](https://mrramoun.github.io)
 
-# Project Name
+# Ramoun's Portfolio
 
-> One paragraph statement about the project.
+> A Simple Portfolio App for a Web developer.
 
 ![screenshot](images/app_screenshot.png)
```

Comment:
I know you did a good effort doing your description for this project, but I know you can do a more impressive README file, if you want to have a reference, 3 lines or more would be nice, just try to think about what you did, what kind of technologies or techniques you used, what is the scope for this project, and so on.

- [ ] Improve your description for your project.